### PR TITLE
APIGOV-27683 environment filtering

### PR DESCRIPTION
**Pull request by alrosca**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Updated Apigee X documentation to reflect the Apigee Agents environment filtering

## Deploy preview link

http://pr-439.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_apigee_x/

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | aaf91b3 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-439.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md [hunk 1]
```diff
@@ -141,6 +141,9 @@ The installation procedure will prompt for the following:
    * **Project ID**: the Project ID for your Google Cloud Platform project
    * **Developer Email**: the email address of a developer, defined in Apigee, that will be given ownership of all Applications
    * **Client Email**: the email address, principal name, for the service account in GCP that has the role to discovery Apigee resources
+   * **Environment**: filter proxies (discovery)/filter metrics (traceability)
+     * discovery: set to discover proxies that are deployed only in a specific environment, if not set discover all environments
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_apigee_x/_index.md:145]: : set to discover proxies that are deployed only in a specific environment.  If not set, proxies in all environments will be discovered.

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md [hunk 2]
```diff
@@ -141,6 +141,9 @@ The installation procedure will prompt for the following:
...
+     * traceability: set to filter metrics for proxies deployed in a specific environment, if not set gather metrics from all environments
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_apigee_x/_index.md:146]: : set to filter metrics for proxies deployed in a specific environment.  If not set, metrics will be gathered for all environments

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md [hunk 3]
```diff
@@ -141,6 +141,9 @@ The installation procedure will prompt for the following:
    * **Project ID**: the Project ID for your Google Cloud Platform project
    * **Developer Email**: the email address of a developer, defined in Apigee, that will be given ownership of all Applications
    * **Client Email**: the email address, principal name, for the service account in GCP that has the role to discovery Apigee resources
+   * **Environment**: filter proxies (discovery)/filter metrics (traceability)
+     * discovery: set to discover proxies that are deployed only in a specific environment. If not set, proxies in all environments will be discovered.
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_apigee_x/_index.md:145]: sorry @alrosca , one small change :) .. in keeping consistent, you can remove the period after discovered.

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md [hunk 4]
```diff
@@ -141,6 +141,9 @@ The installation procedure will prompt for the following:
    * **Project ID**: the Project ID for your Google Cloud Platform project
    * **Developer Email**: the email address of a developer, defined in Apigee, that will be given ownership of all Applications
    * **Client Email**: the email address, principal name, for the service account in GCP that has the role to discovery Apigee resources
+   * **Environment**: filter proxies (discovery)/filter metrics (traceability)
+     * discovery: set to discover proxies that are deployed only in a specific environment. If not set, proxies in all environments will be discovered
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_apigee_x/_index.md:145]: Suggest adding example of the filter like with do for the APIs

**Comment by lbadenhop** [comment]
@alrosca, please see Robert's comment

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md (hunk 4 above)
Comment (jcollins-axway) [reply content/en/docs/connect_manage_environ/connect_apigee_x/_index.md]: added an example, @robertengaxway does this work?

**Review by robertengaxway — APPROVED** [review]
approved

**Comment by lbadenhop** [comment]
@alrosca, are you ok with the changes?

**Review by jcollins-axway — APPROVED** [review]

**Pull request merged**